### PR TITLE
Ghameerah/add styling

### PR DESCRIPTION
**Pull request by ameerrah9**
Push to develop instead main

### CODE REVIEW

**Comment by royanger** [comment]
You can go into the Settings section, the Branches. Add a branch protection rule for main that requires a PR an at least one approval before merging. Also check off 'Include Administrators' near the bottom.

**Review by gsaraa — APPROVED** [review]

**Pull request merged**
